samples/plugins/sqlite_extract/dialect.xml

```diff
@@ -0,0 +1,5 @@
+<dialect name='SQLiteODBCExtract'
+         class='sqlite_extract'
+         version='18.1'
+         base='CDataExtractDialect'>
```

Comment:
We should consider adding a DefaultExtractDialect and using it in this sample going forward

Reply:
Sounds like a good idea